docs/SUMMARY.md

```diff
@@ -57,5 +57,5 @@
 * [💻 Key Concepts](resources/README.md)
   * [JavaScript](resources/javascript-basics.md)
   * [React](resources/react-basic.md)
-* [🤔 FAQ](faq.md)
+* [🤔 FAQ](faq/faq.md)
```

Comment:
Don't you have to change this link to `faq/README.md`

Reply:
Already done